src/components/UI/Footer/helpers.tsx

```diff
@@ -1,18 +1,30 @@
 import React from "react";
-import { FooterColumnItemProps } from "./types";
+import { FooterColumnData, FooterColumnItemProps, FooterColumnProps } from "./types";
 
-export const defaultRender = ({ label, to }: FooterColumnItemProps): React.ReactElement => {
+const defaultRender = ({ label, to }: FooterColumnItemProps): React.ReactElement => {
   return (
     <a className="text-cloud-500" href={to}>
       {label}
     </a>
   );
 };
 
-export const Bottom = ({ copyright }: { copyright: string }): React.ReactElement => {
+export const mapper = (item: FooterColumnData, index: number): React.ReactElement => {
+  const { render = defaultRender } = item;
+  return (
+    <div key={`row-${index}`} className={"pb-3"}>
+      {render({ ...item })}
+    </div>
+  );
+};
+
+export const Bottom = ({ copyright, data }: { copyright: string; data: FooterColumnProps[] }): React.ReactElement => {
```

Comment:
even though FooterColumnProps has the shape that you want for the data, its better to redefine a new abstraction for the data for bottom. or rename FooterColumnProps into a more gnerci term for both the columns and bottom components to use.

Reply:
ok updated